src/Manifest/ManifestLoader.php

```diff
@@ -19,10 +19,16 @@ class ManifestLoader
 	 */
 	private $directoryProvider;
 
+	/**
+	 * @var ?ManifestMapper
```

Comment:
I think the code would be more streamlined if the mapper wasn't nullable. This package could by default use a `ManifestPluginMapper` which would just `return $manifest` in the `map()` method. Could you implement that please?

Reply:
OK, will make default Identity mapper.